File: 1323-maximum-69-number/1323-maximum-69-number.java
class Solution {
    public int maximum69Number (int num) {
        StringBuilder numSb = new StringBuilder();
        numSb.append(num);
        
        for(int i = 0;i<numSb.length(); ++i)
        {
            if(numSb.charAt(i) == '6'){
                numSb.setCharAt(i,'9');
                break;
            }
        }
        return Integer.parseInt(numSb.toString());
    }
}

File: 1323-maximum-69-number/1323-maximum-69-number.cpp
class Solution {
public:
    int maximum69Number (int num) {
        string numString = to_string(num);
        for(auto &curChar : numString){
            if( curChar == '6'){
                curChar = '9';
                break;
            }
        }
        return stoi(numString);
        
    }
};